vtr_flow/arch/titan/README.txt

```diff
@@ -0,0 +1,48 @@
+This folder contains architecture files for use with Titan.
```

Comment:
Can you keep this file as an `rst` file?

Reply:
Thank you for reviewing. I've changed it back to a rst file.